pkg/sendbuffer/sendbuffer.go

```diff
@@ -16,7 +16,7 @@ type sender interface {
 }
 
 var (
-	defaultMaxSize     = 128 * 1024
+	defaultMaxSize     = 512 * 1024
```

Comment:
too much?

Reply:
What's the unit?

Reply:
bytes, I updated all the size vars to have bytes in their name